client/landing/stepper/declarative-flow/new-hosted-site-flow.ts

```diff
@@ -226,6 +232,48 @@ const hosting: Flow = {
 			[]
 		);
 	},
+	useAssertConditions(): AssertConditionResult {
```

Comment:
I feel like the `useSideEffect` hook would be more appropriate than `useAssertConditions`. The assert conditions hook is all about returning an error, but that's really only a secondary concern to us. The primary thing we're trying to do is run a side-effect (call `window.location.assign()`) on certain conditions. So it feels like adding another `useEffect()` to `useSideEffect()` would make more sense.